platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/functional/SOCPeopleActivityCommentTestIT.java

```diff
@@ -0,0 +1,367 @@
+    HomePagePlatform homePagePlatform;
+    ConnectionsManagement connectionsManagement;
+    AddUsers addUsers;
+
+    @BeforeEach
+    public void setupBeforeMethod() {
+        info("Start setUpBeforeMethod");
+        navigationToolbar = new NavigationToolbar(this);
+        addUsers = new AddUsers(this);
+        manageLogInOut = new ManageLogInOut(this);
+        activityStream = new ActivityStream(this);
+        homePagePlatform = new HomePagePlatform(this);
+        connectionsManagement = new ConnectionsManagement(this);
+        manageLogInOut.signInCas(DATA_USER1, "gtngtn");
+    }
+
+    /**
+     * <li> Case ID:122799.</li>
+     * <li> Test Case Name: Comment on your activity.</li>
+     * <li> Pre-Condition: </li>
+     * <li> Post-Condition: </li>
+     * Step Number: 1
+     * Step Name: -
+     * Step Description:
+     * Step 1: Add new activities
+     * Input Data:
+     * - Sign in system
+     * - Select Activities page on User Toolbar portlet in the upper right corner of the screen
+     * - Select activity in the left pane
+     * - Enter some text into text box
+     * - Click on [Share] button
+     * Expected Outcome:
+     * Add an activity successfully
+     * - This activity is added into users activities list.User who is in your contact, can view your active on his/her activity list
+     * Step number: 2
+     * Step Name: -
+     * Step Description:
+     * Step 2: Show comment form
+     * Input Data:
+     * - Click on Comment link under the activity
+     * Expected Outcome:
+     * Show a text box allow user add comment for activity
+     * Step number: 3
+     * Step Name: -
+     * Step Description:
+     * Step 3: Add a comment
+     * Input Data:
+     * - Enter some text into text box comment
+     * - Click on Comment button
+     * Expected Outcome:
+     * Add a comment successfully. Other user can view activity and comment of one. With each comment, some information is display:
+     * - Avatar of user comment
+     * - Name of user comment
+     * - Content of comment
+     * - Time comment is posted
+     */
+    @Test
+    public void test01_CommentOnYourActivity() {
+        info("Test 1: Comment on your activity");
+        info("Add new user");
+        String username1 = "usernamea" + getRandomString();
+        String email1 = username1 + "@gmail.com";
+        String password = "123456";
+        info("Add new user");
+        navigationToolbar.goToAddUser();
+        addUsers.addUser(username1, password, email1, username1, username1);
+        manageLogInOut.signIn(username1, password);
+
+        String activity1 = "activity1" + getRandomNumber();
+        activityStream.addActivity(activity1, "");
+        String comment = "comment" + getRandomNumber();
+        activityStream.commentActivity(activity1, comment);
+        $(byText(activity1)).parent().parent().find(byText(comment)).waitUntil(Condition.visible, Configuration.timeout);
+        activityStream.deleteactivity(activity1);
```

Comment:
deleteactivity function must be updated with the new behavior

Reply:
it's already updated with the new behavior. i used the last one created